src/js/pages/catalog/__tests__/PackageDetailTab-test.js

```diff
@@ -239,16 +239,5 @@ describe("PackageDetailTab", function() {
       thisInstance.state.isLoadingSelectedVersion = true;
       expect(thisInstance.render()).toMatchSnapshot();
     });
-
-    it("renders an infobox, when a dependency of the service has not been installed yet.", function() {
```

Comment:
I spend a lot of time trying to get this to work. It was failing because of i18n._
https://github.com/dcos/dcos-ui/pull/4076/files#diff-ef04d5bf3205041917c17a93e5389d86R626

In the end I decided to delete this and add an integration test instead. I considered deleting this entire file.